### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kaosarbitrium**
Might want to work on a better project description in the future(?)
Have a basic "Here's how you install and get started" done in the "Install" and "Documentation/Wiki" sections, will need to be expanded upon in the future I'm sure.
Credits are far from complete. Will need to find a way to link to and credit the artists and writing teams. For now, I've just linked to everyone(?) with access to the project on GitHub.
License is currently undetermined to my knowledge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,18 +8,35 @@ Installation
 ============
 
 describe how to install renpy and/or any other tools we may use in the future.
```
Comment (PradheepKethiReddy) [inline README.md]: Remove this line

File: README.md [hunk 2]
```diff
@@ -8,18 +8,35 @@ Installation
...
 Wiki/Documentation
 ==================
 
-decide to add documentation in a google drive or make a wiki and use here.
+Ren'Py release used for development: https://www.renpy.org/release/7.4.7
+QuickStart guide for Ren'Py: https://www.renpy.org/doc/html/quickstart.html
+Ren'Py Reference Manual: https://www.renpy.org/doc/html/
 
-I personally would like to keep documentation internally since we'd likely have to link to other groups's work such as their art folder and such. Much easier imo to link around in google docs.
+For access to internal developer notes and discord, message one of the project managers.
```
Comment (PradheepKethiReddy) [inline README.md]: Capitalize Discord in this line

File: README.md [hunk 3]
```diff
@@ -8,18 +8,35 @@ Installation
...
 
 Credits
 =======
 
-put credits here.
+put credits here.
```
Comment (PradheepKethiReddy) [inline README.md]: Remove this line

File: README.md [hunk 4]
```diff
@@ -8,18 +8,35 @@ Installation
...
-I personally would like to keep documentation internally since we'd likely have to link to other groups's work such as their art folder and such. Much easier imo to link around in google docs.
+For access to internal developer notes and discord, message one of the project managers.
 
 Credits
 =======
 
-put credits here.
+put credits here. 
+
+https://github.com/Laytoncie      - Project Owner/Developer
```
Comment (PradheepKethiReddy) [inline README.md]: Laytoncie is not a Project Owner. Redefine this line as

https://github.com/Laytoncie            - Lead Developer

File: README.md [hunk 5]
```diff
@@ -8,18 +8,35 @@ Installation
...
+https://github.com/git-dead       - Project Owner/Developer
+https://github.com/Renaulde       - Project Owner/Developer
+https://github.com/Kaosarbitrium  - Project Developer
+https://github.com/LewRit         - Project Developer
+https://github.com/MCBama         - Project Developer
+https://github.com/Spid3rBite     - Project Developer
+https://github.com/ThatSkyGuy     - Project Developer
 
 License
 =======
```
Comment (PradheepKethiReddy) [inline README.md:51]: Look in to Apache vs MIT license for the future so we can discuss what might be the best fit.

Comment (Kaosarbitrium) [reply README.md:51]: I'll try and deep dive the two, see what's what. 
It'll take me a minute, but I'll come back with a pro-con report for you.
OpenGL Version [x] is also a common license, though I think that's more for Open Source projects. Want me to bother looking into those?

Comment (PradheepKethiReddy) [reply README.md:51]: Due to the openness of this project I would appreciate a bit of information on the OpenGL license you've mentioned.

Also do not feel as though it needs to be a 'professional' pro-con list. This will be something we will discuss during the next programming meeting since license information is something important for all programmers to know.

File: README.md (hunk 2 above)
Comment (PradheepKethiReddy) [reply README.md]: Sentence should probably include 'the' before 'Discord' but that is an issue for the next PR for the README.

**Comment by Kaosarbitrium** [comment]
Here's a decent pro-con report on the licenses, with links to the full texts.

MIT 2.0 License:
	An extremely short, easy to comply to license. Provides protection for Copyright and Copyright holding, but does not mention protection for pattents and pattent litiagtion.
	(It's only five paragraphs long, three of which are single lines.)
	
Apache 2.0 License: 
	Is a lot more specific with deffinitions and credit protection, also has *explicit* protection for patents used in the software and resulting patent litigation.
	Also requires explicit statement in derivitive works of what files are from our source, and if they were edited or not.
	
Both allow for:
	Commercial Use (Selling the Source Code/Software of the Project, providing all conditions for Distribution are met.)
	Modification (The changing of a local copy of the Source Code's/Software's content or contributing to the project's Source Code/Software content.)
	Distribution (Sharing of Source Code or an encapuslated Object of the Software like an .exe file, per specific notices and conditions)
	Sublicensing (Licensing a derivitive work under a new MIT/Apache License automatically)
	Private Use (Such as on a personal machine only)
	
	
Neither allow for:
	License holder(us) to be held liable for Licensors/Sublicensors use of the software, nor do the provide any kind of warranty for their useing our code/project/object.
	Neither License forces derrivitive code to be of the same License, nor do they provide any limitation on how "open" the software's Source Code must be. (GNU General Public Lisences do.)
	
Apache specifically does not allow for:
	Licensors/Sublicensors to use the Contributors Names, Trademarks, or Logos beyond giving appropriate credit
	
	
TL;DR:

	MIT:
		Extremely simple, basically just says "Credit where you got your code from. They're not liable for what you do, tho."
		Would work just fine for what we're doing here, but it does lack some extra legal nicities like extensive deffinitions and pattent/trademark protections.
		MIT License is a recommended pick for this project.
		
	Apache:
		Also, quite simple for a legal document. Basically a more robust, well defined "version" of the MIT license that allows us to keep personal protections, protections of the project's IP, and forces credit to be given more explicitly rather than a simple "I used MIT Licensed code in this file/project."
		While the MIT License should work out fine for the project, this would be a better option for if we have any plans to sell the finished product, or even to put it up for free on stores like Steam, as it'd be easier to take down reposts/clones via things like DMCA.
		Apache 2.0 License is my MOST recommended pick for this project.
		
	GNU General Public License:
		Would be the most restrictive on this project and any derrivitive works. Would force the Source Files/Code to be publically available at all times, and would force any derrivitive works to use the same license we did.
		While not technically a problem, that would hinder marketability should we plan to sell the project, and would be a hassle for anyone who might want to make things like Mods or Expansions for the game, or even use it as a base for their own game.
		GNU General Public License is my LEAST recommended pick for this project.
		
		
Links to Licenses:

Apache V2.0 License: 
	https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)#fulltext
MIT License: 
	https://tldrlegal.com/license/mit-license#fulltext
GNU General Public License(Multiplse Versions Available)
	https://tldrlegal.com/license/gnu-lesser-general-public-license-v3-(lgpl-3)#fulltext
